test/framework/silent_kubectl/client.go

```diff
@@ -0,0 +1,44 @@
+// nolint:all  // code here is extracted and revised from github.com/gruntwork-io/terratest
```

Comment:
why do you extracting the code instead of just adding the `github.com/gruntwork-io/terratest` as dependency?

Reply:
I duplicated these functions to remove the redundant and verbose logging. ("Configuring kube client using..." blabla, this logging is polluting the test output very much since I need to poll the cluster)
I'll remove these duplications once `terratest` supports configure these verbosity. (Charly has a PR to do it, I'll find some time and help it merged)

Reply:
And that's why I call it "silent kubectl", because the one from the module talks too much.